src/OrchardCore.Modules/OrchardCore.Sitemaps/ViewModels/CustomUrlSitemapSourceViewModel.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+using OrchardCore.Sitemaps.Models;
+
+namespace OrchardCore.Sitemaps.ViewModels
+{
+    /// <summary>
+    /// A sitemap source for managing custom url.
+    /// </summary>
+    public class CustomUrlSitemapSourceViewModel
+    {
+        /// <summary>
+        /// the custom url.
```

Comment:
If you want to add a summary to each property, please start with `Gets or sets ...`